coverage/html.py

```diff
@@ -412,9 +421,14 @@ def write_html_file(self, ftr: FileToReport, prev_html: str, next_html: str) ->
                 )
             ldata.css_class = ' '.join(css_classes) or "pln"
 
+        di = file_data.__dict__
+        if contexts_json == "{}":
+            di["contexts_json"] = None
```

Comment:
Wouldn't it just be easier to use:
```python
file_data["contexts_json"] = None
```
and then not have to create the `di` variable?